index.js

```diff
@@ -0,0 +1,19 @@
+// Первый аргумент - массив, второй - колбек-функция, которая применяется на массив
+// Колбек-функция применяется на каждый элемент массива и проверяет условие
+// Если хотя бы один раз колбек-функция вернула true, то any возвращает true
+// Иначе - false
+// Если колбек-функция не была передана, то из any возвращаем true, 
+// если в массиве есть хотя бы один элемент, который дает true,
+// если все значения в массиве дают false, то возвращаем false
+
+//O(n)
+function any(array, collback = (x) => !!x === true) {
```

Comment:
(x) => !!x === true    => (x) => !!x

Reply:
We don't check usually equality to boolean value because !! does it

Reply:
Understood